### DISCUSSION TITLE
Just asking

### DISCUSSION THREAD

**Discussion by eilonn** [ndeadly/MissionControl#617] [Q&A]
do u know if xLizzard XXZ-J5-01 works with the addon?

**Comment by ndeadly** [comment]
Probably not, given I've never heard of it. Mission Control needs to have support for a given controller explicitly added. But if it's a clone of something else it might. Best thing is just to try it. If it connects to the console at all it should be pretty easy to add support.

**Reply by eilonn** [reply]
> Probably not, given I've never heard of it. Mission Control needs to have support for a given controller explicitly added. But if it's a clone of something else it might. Best thing is just to try it. If it connects to the console at all it should be pretty easy to add support.

it doesnt connect, but its a Bluetooth controller, not LE

**Reply by ndeadly** [reply]
Download and install [this](https://github.com/ndeadly/MissionControl/files/11311080/MissionControl-0.9.1-debug-logging-05bbc5b.zip) version and reboot. It will create a log in your sd root (`mc-mitm.log`). Post it here and I'll take a look to see if there's any evidence of it trying to pair with the console.
